testing/infra/terraform/modules/ec_deployment/scripts/enable_features.tftpl

```diff
@@ -14,19 +14,19 @@ if [[ "$${POLICY}" == *"$${NOT_FOUND_MSG}"* ]]; then
     exit 0
 fi
 
-echo $${POLICY} | grep '"expvar_enabled":{"type":"bool","value":true}' > /dev/null 2>&1 && EXPVAR_ENABLED=true
-echo $${POLICY} | grep '"pprof_enabled":{"type":"bool","value":true}' > /dev/null 2>&1 && PPROF_ENABLED=true
-if [[ $${EXPVAR_ENABLED} || $${PPROF_ENABLED} ]] ; then
-    echo "expvar or pprof already enabled"
-    exit 0
-fi
+# Abort early if features are already set as desired
```

Comment:
[to reviewer] unless there's a need to avoid making a no-op agent policy change, I don't see why we need this error-prone check. Before this PR, the check was useless anyway as the grep content `{"type":"bool","value":true}` had out of order JSON keys and it always failed.

Reply:
I'm inclined to remove it